### PR TITLE
Use assertj fluent style in flowable-engine.

### PR DESCRIPTION
**Pull request by dbmalkovsky**
All test files in the `org.flowable.engine.test.api.identity` package are included.

Continuing quest to use only a single style in each file and in the module.

### CODE REVIEW

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java [hunk 1]
```diff
@@ -1,435 +1,384 @@
...
+                .isInstanceOf(RuntimeException.class);
+
+        identityService.deleteGroup(sales.getId());
+        identityService.deleteUser(johndoe.getId());
+    }
+
+    @Test
+    public void testSaveGroupNullArgument() {
+        assertThatThrownBy(() -> identityService.saveGroup(null))
+                .isExactlyInstanceOf(FlowableIllegalArgumentException.class);
```
Comment (filiphr) [inline modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java]: Missing `assertTextPresent`

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java [hunk 2]
```diff
@@ -1,435 +1,384 @@
...
+    }
+
+    @Test
+    public void testSaveUserNullArgument() {
+        assertThatThrownBy(() -> identityService.saveUser(null))
+                .isExactlyInstanceOf(FlowableIllegalArgumentException.class);
```
Comment (filiphr) [inline modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java]: Missing `assertTextPresent`

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java [hunk 3]
```diff
@@ -1,435 +1,384 @@
...
+    }
+
+    @Test
+    public void testFindGroupByIdNullArgument() {
+        assertThatThrownBy(() -> identityService.createGroupQuery().groupId(null).singleResult())
+                .isExactlyInstanceOf(FlowableIllegalArgumentException.class);
```
Comment (filiphr) [inline modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java]: Missing `assertTextPresent`

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java (hunk 1 above)
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java]: Fixed; thanks

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java (hunk 3 above)
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java]: Fixed; thanks

File: modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java (hunk 2 above)
Comment (dbmalkovsky) [reply modules/flowable-engine/src/test/java/org/flowable/engine/test/api/identity/IdentityServiceTest.java]: Fixed; thanks

**Comment by dbmalkovsky** [comment]
Thanks @filiphr; updated and committed to the branch

**Comment by filiphr** [comment]
Thanks for the PR @dbmalkovsky

**Pull request merged**